code/game/objects/items/theft_items.dm

```diff
@@ -0,0 +1,295 @@
+		radiation_pulse(user, 100)
+		playsound(src, 'sound/effects/supermatter.ogg', 50, TRUE)
+		qdel(W)
+		qdel(src)
+
+/obj/item/nuke_core/supermatter_sliver/throw_impact(atom/hit_atom, datum/thrownthing/throwingdatum)
+	if(!isliving(hit_atom))
+		return ..()
+	var/mob/living/victim = hit_atom
+	if(victim.incorporeal_move || victim.status_flags & GODMODE) //try to keep this in sync with supermatter's consume fail conditions
+		return ..()
+	if(throwingdatum?.thrower)
+		var/mob/user = throwingdatum.thrower
+		add_attack_logs(user, victim, "[victim] consumed by [src] thrown by [user] ")
+		message_admins("[src] has consumed [key_name_admin(victim)] [ADMIN_JMP(src)], thrown by [key_name_admin(user)].")
+		investigate_log("has consumed [key_name(victim)], thrown by [key_name(user)]", "supermatter")
+	else
+		message_admins("[src] has consumed [key_name_admin(victim)] [ADMIN_JMP(src)] via throw impact.")
+		investigate_log("has consumed [key_name(victim)] via throw impact.", "supermatter")
+	victim.visible_message("<span class='danger'>As [victim] is hit by [src], both flash into dust and silence fills the room...</span>",\
+		"<span class='userdanger'>You're hit by [src] and everything suddenly goes silent.\n[src] flashes into dust, and soon as you can register this, you do as well.</span>",\
+		"<span class='hear'>Everything suddenly goes silent.</span>")
+	victim.dust()
+	radiation_pulse(src, 500, 2)
+	playsound(src, 'sound/effects/supermatter.ogg', 50, TRUE)
+	qdel(src)
+
+/obj/item/nuke_core/supermatter_sliver/pickup(mob/living/user)
+	..()
+	if(!isliving(user) || user.status_flags & GODMODE) //try to keep this in sync with supermatter's consume fail conditions
+		return FALSE
+	user.visible_message("<span class='danger'>[user] reaches out and tries to pick up [src]. [user.p_their()] body starts to glow and bursts into flames before flashing into dust!</span>",\
+			"<span class='userdanger'>You reach for [src] with your hands. That was dumb.</span>",\
+			"<span class='hear'>Everything suddenly goes silent.</span>")
+	radiation_pulse(user, 500, 2)
+	playsound(src, 'sound/effects/supermatter.ogg', 50, TRUE)
+	user.dust()
+
+/obj/item/nuke_core_container/supermatter
+	name = "supermatter bin"
+	desc = "A tiny receptacle that releases an inert hyper-noblium mix upon sealing, allowing a sliver of a supermatter crystal to be safely stored."
+	var/obj/item/nuke_core/supermatter_sliver/sliver
+
+/obj/item/nuke_core_container/supermatter/Destroy()
+	QDEL_NULL(sliver)
+	return ..()
+
+/obj/item/nuke_core_container/supermatter/load(obj/item/retractor/supermatter/T, mob/user)
+	if(!istype(T) || !T.sliver)
+		return FALSE
+	T.sliver.forceMove(src)
+	sliver = T.sliver
+	T.sliver = null
+	T.icon_state = "supermatter_tongs"
+	T.item_state = "supermatter_tongs"
+	icon_state = "supermatter_container_loaded"
+	to_chat(user, "<span class='warning'>Container is sealing...</span>")
+	addtimer(CALLBACK(src, .proc/seal), 10 SECONDS)
+	return TRUE
```

Comment:
Why the `return TRUE`/`return FALSE` here? `load()` isn't ever used in an if statement so it doesn't matter what's being returned.
Same on the parent proc too of course.

Reply:
Removing